slasher/detection/detect.go

```diff
@@ -31,10 +35,12 @@ func (ds *Service) detectAttesterSlashings(
 		switch result.Kind {
 		case types.DoubleVote:
 			slashing, err = ds.detectDoubleVote(ctx, att, result)
+			logrus.Debugf("Detected a possible double vote for val idx %d", valIdx)
```

Comment:
Are we gonna keep this logs? Maybe we should remove them since we know detection works now.